internal/services/kusto/kusto_database_script_resource.go

```diff
@@ -109,12 +123,22 @@ func resourceKustoDatabaseScriptCreateUpdate(d *pluginsdk.ResourceData, meta int
 
 	parameters := kusto.Script{
 		ScriptProperties: &kusto.ScriptProperties{
-			ContinueOnErrors:  utils.Bool(d.Get("continue_on_errors_enabled").(bool)),
-			ForceUpdateTag:    utils.String(forceUpdateTag),
-			ScriptURL:         utils.String(d.Get("url").(string)),
-			ScriptURLSasToken: utils.String(d.Get("sas_token").(string)),
+			ContinueOnErrors: utils.Bool(d.Get("continue_on_errors_enabled").(bool)),
+			ForceUpdateTag:   utils.String(forceUpdateTag),
 		},
 	}
+	if ScriptURL, ok := d.GetOk("url"); ok {
+		parameters.ScriptURL = utils.String(ScriptURL.(string))
+	}
+
+	if ScriptURLSasToken, ok := d.GetOk("sas_token"); ok {
+		parameters.ScriptURLSasToken = utils.String(ScriptURLSasToken.(string))
+	}
+
+	if ScriptContent, ok := d.GetOk("script_content"); ok {
```

Comment:
The local variable should be in lower camel case

Reply:
Done